cloudfront/lambdas/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lambdas",
+  "version": "1.0.0",
+  "main": "index.js",
+  "engines": {
+    "node": "12.*.*"
+  },
+  "license": "MIT",
+  "scripts": {
+    "test": "yarn jest",
+    "build": "tsc -p tsconfig.build.json && (cd dist && zip -r lambda.zip .)",
+    "deploy": "yarn build && node deploy",
+    "dockerLoginLocal": "aws ecr get-login-password --region eu-west-1 --profile platform | docker login --username AWS --password-stdin 760097843905.dkr.ecr.eu-west-1.amazonaws.com",
+    "dockerBuildLocal": "yarn dockerLoginLocal && docker build . -t weco_cf_lambdas",
+    "dockerTestLocal": "yarn dockerBuildLocal && docker run weco_cf_lambdas yarn test",
+    "dockerDeployLocal": "yarn dockerBuildLocal && docker run weco_cf_lambdas -v ~/.aws:/root/.aws yarn deploy"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.71",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.25",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.5.0",
+    "tsc": "^1.20150623.0",
```

Comment:
I assume this is a mistake

Reply:
Which what?

Reply:
tsc dependency shouldn't be there (tsc is part of typescript) https://www.npmjs.com/package/tsc